rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp

```diff
@@ -85,8 +85,6 @@ class MultiThreadedExecutor : public rclcpp::Executor
   size_t number_of_threads_;
   bool yield_before_execute_;
   std::chrono::nanoseconds next_exec_timeout_;
-
-  std::set<TimerBase::SharedPtr> scheduled_timers_;
```

Comment:
@ivanpauno @audrow This looks like it breaks ABI. Is it possible to implement this in a way that can be back ported to Foxy? One of the Foxy CI failures looks like it might be an instance of #1212 

http://build.ros2.org/job/Fci__nightly-fastrtps_ubuntu_focal_amd64/193/testReport/junit/(root)/projectroot/gtest_multithreaded__rmw_fastrtps_cpp/

Reply:
I can avoid deleting that line, but the patch will still break ABI because of the addition of:
https://github.com/ros2/rclcpp/blob/3d0f8df8a86d93d362dc79cd7aaa95dcd9f9031b/rclcpp/include/rclcpp/timer.hpp#L260

I don't see how to avoid breaking ABI.